concepts/ranges/about.md

```diff
@@ -3,6 +3,7 @@
 [Ranges][range] represent a sequence of one or many consecutive integers. They:
 
 - Are created using the [`..` operator][range-operator] and can be both ascending or descending.
+- Their default step is `1`, but can be modified using the [`..//` operator][range-operator-with-step] (since Elixir 1.12).
```

Comment:
Adding info about a new feature, but only to `about.md` because it doesn't affect how you solve the exercise.